src/lib.rs

```diff
@@ -41,7 +41,7 @@
 /// let text_re = Regex::new(r".+?\b").unwrap();
 /// let matcher = FMBuilder::new("$1 $1")
 ///                         .unwrap()
-///                         .name_matcher(Some((ptn_re, text_re)))
+///                         .name_matcher((ptn_re, text_re))
```

Comment:
Would it be more user friendly for this function to accept two values, rather than a pair?

Reply:
Yes, probably. It's a mildly more annoying API transition, but if you think it's worth it, I'm happy to make that change.

Reply:
Cool. The API already changed here, so you have nothing to lose.

Reply:
Addressed in https://github.com/softdevteam/fm/pull/24/commits/3f30317a263ba9f1498dde80649b57172c0b6c82.